jerry-core/ecma/base/ecma-globals.h

```diff
@@ -1904,6 +1906,16 @@ typedef struct
   uint8_t element_size; /**< element size based on [[TypedArrayName]] in Table 49 */
 } ecma_typedarray_info_t;
 
+#if ENABLED (JERRY_BUILTIN_BIGINT)
+/**
+ * Checks whether a given typedarray is BigInt type or not.
+ *
+ * Note: this macro should be updated when new types are added
```

Comment:
This note should be in above `ECMA_BIGINT64_ARRAY` in `ecma_typedarray_type_t`. Otherwise people will not notice this.